src/App.tsx

```diff
@@ -14,11 +14,109 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+interface State {
+  selectedGood: string,
+}
+
+class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
+
+  addActiveClass = 'list_item--active';
+
+  render() {
+    return (
+      <>
+        <div className="App">
+          <h1>
+            {this.state.selectedGood
+              ? (`Selected good: - ${this.state.selectedGood}`)
```

Comment:
You can destruct to not write all the time `this.state...`, You will just use your variable in every place instead of `this.state...`